app/src/main/java/com/kickstarter/viewmodels/ChangeEmailViewModel.kt

```diff
@@ -38,6 +42,15 @@ interface ChangeEmailViewModel {
         /** Emits a string to display when email update fails.  */
         fun error(): Observable<String>
 
+        /** Emits a boolean to determine if the user is a creator. */
+        fun isCreator(): Observable<Boolean>
```

Comment:
But what does being a creator signify in this viewmodel? In the activity, it seems like we show some text so maybe it'd be like `shouldShowBlahBlahText`